js/whitebit.js

```diff
@@ -615,11 +706,437 @@ module.exports = class whitebit extends Exchange {
+        //     "left": "0.001",                   // if order not finished - rest of amount that must be finished
+        //     "dealFee": "0",                    // fee in money that you pay if order is finished
+        //     "price": "40000",                  // price
+        //     "activation_price": "40000"        // activation price -> only for stopLimit, stopMarket
+        // }
+        // For Closed Orders
+        //
+        //     {
+        //         "market": "BTC_USDT"             // artificial field
+        //         "amount": "0.0009",               // amount of trade
+        //         "price": "40000",                 // price
+        //         "type": "limit",                  // order type
+        //         "id": 4986126152,                 // order id
+        //         "clientOrderId": "customId11",    // custom order identifier; "clientOrderId": "" - if not specified.
+        //         "side": "sell",                   // order side
+        //         "ctime": 1597486960.311311,       // timestamp of order creation
+        //         "takerFee": "0.001",              // maker fee ratio. If the number less than 0.0001 - its rounded to zero
+        //         "ftime": 1597486960.311332,       // executed order timestamp
+        //         "makerFee": "0.001",              // maker fee ratio. If the number less than 0.0001 - its rounded to zero
+        //         "dealFee": "0.041258268",         // paid fee if order is finished
+        //         "dealStock": "0.0009",            // amount in stock currency that finished
+        //         "dealMoney": "41.258268"          // amount in money currency that finished
+        //     },
+        const marketId = this.safeString (order, 'market');
+        market = this.safeMarket (marketId, market, '_');
+        const symbol = market['symbol'];
+        const side = this.safeString (order, 'side');
+        const filled = this.safeString (order, 'dealStock');
+        const amount = this.safeString (order, 'amount');
+        const clientOrderId = this.safeString (order, 'clientOrderId');
+        const price = this.safeString (order, 'price');
+        const activationPrice = this.safeString (order, 'activation_price');
+        const orderId = this.safeString (order, 'orderId', 'id');
+        const type = this.safeString (order, 'type');
+        const dealFee = this.safeString (order, 'dealFee');
+        let fee = undefined;
+        if (dealFee !== undefined) {
+            let feeCurrencyCode = undefined;
+            if (market !== undefined) {
+                feeCurrencyCode = market['quote'];
+            }
+            fee = {
+                'cost': this.parseNumber (dealFee),
+                'currency': feeCurrencyCode,
+            };
+        }
+        const timestamp = this.safeTimestamp (order, 'ctime', 'timestamp');
```

Comment:
It has to be `this.safeTimestamp2` here, if we're checking two fields.
You can test this with `node examples/js/cli whitebit fetchClosedOrders`

Reply:
damn, I must be sleeping 🤦